docs/specs/api.md

```diff
@@ -0,0 +1,56 @@
+# API
+
+Here are the endpoints that are not part of OIDC, but essential to provide some features of Authgear.
+
+## /api/identity/biometric/remove
+
+This endpoint is for removing Biometric identity.
+The user still can remove biometric identity in the settings page.
+
+### Request Schema
+
+```json
+{
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "kid": { "type": "string" }
```

Comment:
kid is still needed here?

Reply:
Yes, in order to support the case that multiple devices have biometric authentication set up.